izer/izer.py

```diff
@@ -502,7 +502,8 @@ def main():
                            f'{pixelcount}) does not match the '
                            f'sum of all pixels ({pixels}) in the layer\'s `in_sequences`.')
                 input_dim[ll] = conf_input_dim[ll]
-        if operator[ll] != op.CONV1D:
+
+        if not (operator[ll] == op.CONV1D or (operator[ll] == op.NONE and input_dim[ll][1] == 1)):
```

Comment:
if operator[ll] != op.CONV1D and (operator[ll] != op.NONE or input_dim[ll][1] != 1):

Reply:
Changed